js/components/accessibilityControls.js

```diff
@@ -27,6 +27,7 @@ AccessibilityControls.prototype = {
   cleanUp : function() {
     document.removeEventListener("keydown", this.keyEventDown);
     document.removeEventListener("keyup", this.keyEventUp);
+    document.removeEventListener("deviceorientation", this.controller.handleVrMobileOrientation);
```

Comment:
We shouldn't be removing an event listener that was set up in a different file. We should either add the event listener here or move the removal to the original file.

Reply:
\+